FiveCalls/FiveCalls/Issue.swift

```diff
@@ -28,7 +28,7 @@ struct Issue: Identifiable, Decodable {
     }
     
     var shareURL: URL {
-        return URL(string: String(format: "https://5calls.org/issue/%@/",self.slug))!
+        return URL(string: String(format: "https://5calls.org/issue/%@/",self.slug.trimmingCharacters(in: .whitespacesAndNewlines)))!
```

Comment:
Would be nice to have a unit test to ensure this issue doesn't crop up again in the future.

Reply:
👍 added a test